xivo_dao/resources/user/persistor.py

```diff
@@ -58,14 +59,33 @@ def count_all_by(self, column_name, criteria):
         query = query.group_by(column)
         return query.all()
 
-    def search(self, parameters):
+    def search(self, parameters, is_db_sort_limit=True):
```

Comment:
I would prefer 2 functions instead of a boolean argument in this case.
Arguments: https://martinfowler.com/bliki/FlagArgument.html

Reply:
- I prefer having one function that does one thing. In this case, the function does two kinds of sorting, which calls for 2 different functions in my mind.
- I don't find the parameter very self-descriptive at first glance. It also reveals the internal workings of the function, instead of specifying the expected result, i.e. a [collated](https://en.wikipedia.org/wiki/Collation) result. My proposition:
```
def search(...):
def search_collated(...):
```
- Also, I think we can avoid code duplication by making `search_collated` use `search()`.

Reply:
@sduthil code updated, let me know if you're agree or not with what I did.

Reply:
Yes, that's perfect :) Thanks!